include/powerloader/enums.hpp

```diff
@@ -15,6 +15,22 @@ namespace powerloader
         // Want: S3, OCI
     };
 
+    enum CompressionType
```

Comment:
I moved the enums here so that we could have them in the same file. But maybe we don't want that...?

Reply:
If powerloader user dont need them, then maybe moving them into `src/curl_internal.hpp` might work better?